int16 id
sensor_msgs/Image image
---
int16 id
mmdet_deploy/BoundingBoxesInstances bounding_boxes